lib/Dialect/TritonGPU/Transforms/WarpSpecialization/LoadMMASpecialization.cpp

```diff
@@ -422,6 +422,7 @@ LogicalResult PipelinedLoadGroup::lowerLoads(WarpSchedule &schedule,
   for (Operation *asyncUser : distinctAsyncUsers) {
     if (auto mmaOp = dyn_cast<ttng::MMAv5OpInterface>(asyncUser)) {
       mmaOp.addCompletionBarrier(curEmptyBar, b.boolCst(true));
+      mmaOp.setIsSync(false);
```

Comment:
Now that I think about it, there is a use-case for having `is_sync` set to true but with completion barriers on the MMA op: if the loads feeding into an MMA op are warp specialized but the MMA itself is not warp specialized for whatever reason.

Reply:
I saw that happening when the mma is v2. But v2 is always sync.

I haven't seen a case where mmav5 fails to get specialized and put into the default partition.